test/test_functional.py

```diff
@@ -192,6 +192,35 @@ def test_istft_of_sine(self):
         self._test_istft_of_sine(amplitude=99, L=10, n=7)
 
 
+class TestDeltas(unittest.TestCase):
```

Comment:
@zhangguanheng66 recommends merging this class with the one above

PS the test file should be reorganized a bit too :)

Reply:
yeap, we don't need a separate class to cover the dalta func.